plugins/builtin/strategy/target-value/plugin/plugin.go

```diff
@@ -43,6 +51,27 @@ type StrategyPlugin struct {
 	logger hclog.Logger
 }
 
+// scaleDirection is used to indicate if the resulting count should increase
+// (scale up), descrease (scale down), or stay the same (none).
+type scaleDirection int8
+
+const (
+	scaleDirectionUp scaleDirection = iota
+	scaleDirectionDown
+	scaleDirectionNone
```

Comment:
any particular reason that "up" is the zero?
i don't think it causes any problems, but it will probably keep me up at night.
https://play.golang.org/p/Gk2fZilqlLI